src/control/lib/hardware/ucx/bindings.go

```diff
@@ -203,7 +203,7 @@ import (
 func Load() (func(), error) {
 	ucsHdl, err := openUCS()
 	if err != nil {
-		return nil, errors.Wrap(err, "loading libucs")
+		return nil, errors.Wrap(err, "loading libucs.0")
```

Comment:
This is just the error message.  I think you need to also update https://github.com/daos-stack/daos/blob/1a0b8ecb080003f3ade9779e1b9133c99d7cec8d/src/control/lib/hardware/ucx/bindings.go#L228 to append the .1 there also.  Seems in fact that given that this is referenced in 3 places, it needs a single definition that can be de-referenced instead of being duplicated 3 times.